services/registration/src/routes/application.ts

```diff
@@ -236,7 +236,7 @@ applicationRouter.route("/:id/actions/submit-confirmation").post(
       throw new BadRequestError("No confirmation branch is selected.");
     }
 
```

Comment:
Can you also add a check here to make sure a user is accepted to be able to submit a confirmation?